src/validators/bartio/defaultValidatorList.json

```diff
@@ -1,6 +1,14 @@
 {
   "name": "Berachain Bartio Validator List",
   "validators": [
+    {
+      "id":"4D2750382BD203C93BBFFD7256197CD71CB10493",
+      "logoURI":"https://www.puffpaw.xyz/apple-touch-icon.png ",
```

Comment:
I uploaded the logo on our CDN, please replace it with the following: `https://res.cloudinary.com/duv0g402y/image/upload/v1732285700/src/validators/bartio/yambsuta1qibwyabprys.png`